### PR TITLE
CRAN release documentation update

### LINKED ISSUE #81: CRAn Release Documentation Updates
### Please select a category the issue is focused on?

_No response_

### Let us know where something needs a refresh or put your idea here!

- README.Rmd
-News.md

### PR DESCRIPTION
**Pull request by rammprasad**
Documentation updates

closes #81

### CODE REVIEW

**Comment by rammprasad** [comment]
@ramiromagno @galachad - This PR is ready for your review. Let's merge it before we create the release candidate. If there are any pipeline errors, please feel free to fix them.

**Review by ramiromagno — APPROVED** [review]
Hi @rammprasad and @galachad: I've made a few changes:

- Whitelisted some acronyms that got introduced with the new docs
- Fixed the `VignetteIndexEntry` value to match the vignette title in study_sdtm_spec.Rmd
- Fixed the Slack URL that was incomplete in README.md
- Added my name and affiliation to the authors' section to pkgdown configuration for side nav bar hyperlinking
- Rerun current package state in linux, macos, macos-arm64, windows VMs with rhub which seems to be passing fine.

File: _pkgdown.yml [hunk 1]
```diff
@@ -57,3 +57,9 @@ reference:
 - title: Package global state
   contents:
   - clear_cache
+
+authors:
+  Ramiro Magno:
+    href: https://www.pattern.institute/team/rmagno/
+  Pattern Institute:
+    href: https://www.pattern.institute
```
Comment (rammprasad) [inline _pkgdown.yml:65]: @ramiromagno - We already have this information in the Description file. Do we need to repeat this information in the _yml file?

Comment (ramiromagno) [reply _pkgdown.yml:65]: Yes, otherwise the authors on the side navigation bar won't be hyperlinked.

Comment (rammprasad) [reply _pkgdown.yml:65]: I don't think any of us are hyperlinked. If we want hyperlinks, we all need to add our information here. Other Pharmaverse packages don't have this either. Would you consider removing it now and adding it back after the release?

Comment (ramiromagno) [reply _pkgdown.yml:65]: I believe it should be at each author's discretion to display hyperlinks to their pages. I guess other authors are welcome to add their links if they wish. I think the question should be posed the other way around: is there any reason not to want links?

File: NEWS.md [hunk 2]
```diff
@@ -1,33 +1,59 @@
...
+    - Import study controlled terminology
+-  Articles
+    - Algorithms
+    - Creating an Events SDTM domain
+    - Creating a Findings SDTM domain
+    - Conditioned Data Frames
+    - Converting dates, times or date-times to ISO 8601
+    - Path to Automation
+
 # sdtm.oak 0.0.0.9005 (development version)
```
Comment (galachad) [inline NEWS.md:29]: I'm wonder if we should remove all these developer versions. In my opinion, everything should be on 0.1

File: _pkgdown.yml (hunk 1 above)
Comment (edgar-manukyan) [inline _pkgdown.yml:65]:
```suggestion
    href: https://www.pattern.institute
  Edgar Manukyan:
    https://www.linkedin.com/in/edgar-manukyan-20987927
```

Comment (ShiyuC) [reply _pkgdown.yml:65]: I appended my profile in Edgar's comment above. 😸

**Review by galachad — APPROVED** [review]
LGTM

File: NEWS.md [hunk 3]
```diff
@@ -1,33 +1,39 @@
...
+    -   `hardcode_ct()`  to process hardcode_ct algorithm
+    -   `assign_datetime()` to process assign_datetime algorithm
+    -   `condition_add()`  to process condition_add algorithm (if/else conditions)
+-  Functions for SDTM derived variables
+    -   `derive_blfl()` to Derive Baseline Flag or Last Observation Before Exposure Flag
+    -   `derive_seq()` to Derive the sequence number (--SEQ) variable
+    -   `derive_study_day()` to Derive study day
+    -   `create_iso8601()` for ISO8601 date, datetime conversion.
+-  Functions to support {sdtm.oak}
+    - ` generate_oak_id_vars()` to derive oak id variables
```
Comment (ShiyuC) [inline NEWS.md:18]:
```suggestion
    - `generate_oak_id_vars()` to derive oak id variables
```
Remove extra space here

**Pull request merged**